### PR TITLE
UI setup

### PR DESCRIPTION
**Pull request by supermanshuvo**
Dear Sir,
All the work is now done. Remove  * from the blade and fix all the problem.
Thank you
Rezaul Karim

### CODE REVIEW

File: routes/web.php [hunk 1]
```diff
@@ -14,5 +14,20 @@
...
+})->name('index');
+Route::get('/login',function(){
+    return view('contact.login');
+})->name('login');
+Route::get('/register', function(){
+    return view('contact.register');
+})->name('register');
+Route::get('/edit', function(){
+    return view('contact.edit');
+})->name('edit');
```
Comment (asifmalik106) [inline routes/web.php:27]: it should be "edit_contact"

File: routes/web.php [hunk 2]
```diff
@@ -14,5 +14,20 @@
...
+Route::get('/add', function(){
+    return view('contact.add');
+})->name('add');
```
Comment (asifmalik106) [inline routes/web.php:30]: it should be "add_contact"

**Review by asifmalik106 — CHANGES_REQUESTED** [review]
Very simple modification, change route names to suggested one. Replace the previous route name from the entire project. Then again initiate a pull request.

File: routes/web.php (hunk 1 above)
Comment (asifmalik106) [reply routes/web.php:27]: or "contact_edit"

File: routes/web.php (hunk 2 above)
Comment (asifmalik106) [reply routes/web.php:30]: or "contact_add"

File: routes/web.php (hunk 1 above)
Comment (supermanshuvo) [reply routes/web.php:27]: name change done and Thank you.

File: routes/web.php (hunk 2 above)
Comment (supermanshuvo) [reply routes/web.php:30]: Done and Thank You

**Pull request closed without merging**